jwst/tweakreg/tweakreg_step.py

```diff
@@ -72,6 +72,16 @@ class TweakRegStep(Step):
         min_gaia = integer(min=0, default=5) # Min number of GAIA sources needed
         save_gaia_catalog = boolean(default=False)  # Write out GAIA catalog as a separate product
         output_use_model = boolean(default=True)  # When saving use `DataModel.meta.filename`
+        abs_minobj = integer(default=15) # Minimum number of objects acceptable for matching when performing absolute astrometry
+        abs_searchrad = float(default=6.0) # The search radius in arcsec for a match when performing absolute astrometry
+        # We encourage setting this parameter to True. Otherwise, xoffset and yoffset will be set to zero.
+        abs_use2dhist = boolean(default=True) # Use 2D histogram to find initial offset when performing absolute astrometry?
+        abs_separation = float(default=0.1) # Minimum object separation in arcsec when performing absolute astrometry
```

Comment:
Separation should be at least 1.5 times the tolerance in order to avoid memory errors in `xyxymatch`. @stsci-hack Why is it 10x smaller here?